Source/Scene/GlobeTranslucencyFramebuffer.js

```diff
@@ -39,110 +30,70 @@ function GlobeTranslucencyFramebuffer() {
 Object.defineProperties(GlobeTranslucencyFramebuffer.prototype, {
```

Comment:
A few of these properties are only here for easy access in `GlobeTranslucencyFramebufferSpec`. Should those be removed? 

Related question - should `FramebufferManager` expose all its attachments as object property getters even if it isn't needed, just so the design is symmetrical for all attachment types?